source/_docs/unsupported-modules-plugins.md

```diff
@@ -352,9 +352,24 @@ Pantheon has tools in place to monitor database queries:
 
 **Solution**: [Upgrade your site's PHP version](/docs/php-versions) to 5.5, 5.6, or 7.0.
 <hr>
+
+### [UNLOQ Two Factor Authentication (2FA)](https://wordpress.org/plugins/unloq/)
+**Issue**: Login page shows an error `This widget does not work on this domain` after deploying plugin to the site. The API key is stored directly in the database, and there is no Dashboard UI section to set this up or change. If you activated the plugin and cloned your database to a different environment, this will be overwritten and would encounter this error.
+
+**Solution**: As a workaround, manually change 'unloq_credentials' key in 'wp_options' table. You can also re-create an application by resetting your plugin installation (deactivate, delete entries etc.,)
+
+**Issue**: When we activate the plugin it wants us to ensure that .htaccess is writeable as an error message
```

Comment:
Let's ship the first issue in this PR, but move discussion on the second issue back to #2998 so we can test the patch

Reply:
Yes and agree for this